docs/UserMenu.md

```diff
@@ -30,16 +30,16 @@ import { UserMenu } from '@brightlayer-ui/react-native-components';
 
 <div style="overflow: auto">
 
-| Prop Name       | Description                                              | Type                          | Required | Default |
-| --------------- | -------------------------------------------------------- | ----------------------------- | -------- | ------- |
-| avatar          | Avatar to display as the menu trigger                    | `JSX.Element`                 | yes      |         |
-| backgroundColor | Background color of the bottom sheet                     | `string`                      | no       |         |
-| fontColor       | Color of text for the bottom sheet header and menu items | `string`                      | no       |         |
-| iconColor       | Color of icons for the bottom sheet menu items           | `string`                      | no       |         |
-| menuItems       | An array of menu items to show in the bottom sheet       | `InfoListItemProps[]`         | yes      |         |
-| menuTitle       | Title to display in the bottom sheet                     | `string`                      | no       |         |
-| menuSubtitle    | Subtitle to display in the bottom sheet                  | `string`                      | no       |         |
-| theme           | Theme value overrides                                    | `$DeepPartial<ExtendedTheme>` | no       |         |
+| Prop Name       | Description                                              | Type                          | Required | Default                         |
+| --------------- | -------------------------------------------------------- | ----------------------------- | -------- | ------------------------------- |
+| avatar          | Avatar to display as the menu trigger                    | `JSX.Element`                 | yes      |                                 |
+| backgroundColor | Background color of the bottom sheet                     | `string`                      | no       |                                 |
```

Comment:
No default background color here?

Reply:
Updated backgroundColor  of bottomsheet as `theme.colors.surface`